insightsengineering/chevron
# complete refactor of postprocess

close #314 

move prune/sort/null_report to post processings

other things changed:
1. move scda to imports (avoid errors when user try to run examples without scda)
2. create data with syn_test_data to increase the speed
3. add helper script for testthat to avoid repeated call of syn_test_data()
4. create null_report object and export it
5. put "no observations.." in content instead of header


decorations, should also go into "post-processings" but currently we have them in main. we can always change it later.
